__test__/extensions/action-handlers/zapier-action.test.js

```diff
@@ -0,0 +1,884 @@
+import {
+  validateActionHandler,
+  validateActionHandlerWithClientChoices
+} from "../../../src/extensions/action-handlers";
+import nock from "nock";
+
+const ZapierAction = require("../../../src/extensions/action-handlers/zapier-action");
+require("../../test_helpers");
+const log = require("../../../src/lib").log;
+
+afterEach(async () => {
+  jest.restoreAllMocks();
+});
+
+describe("zapier-action", () => {
+  let veryFakeOrganization;
+  const OLD_ENV = process.env;
+
+  beforeEach(async () => {
+    process.env = { ...OLD_ENV };
+    jest.spyOn(log, "info");
+    veryFakeOrganization = {
+      id: 9001
+    };
+  });
+
+  afterEach(async () => {
+    process.env = OLD_ENV;
+  });
+
+  it("should pass validation", async () => {
+    expect(() => validateActionHandler(ZapierAction)).not.toThrowError();
+  });
+
+  it("should not report available if minimum environment vars are NOT set", async () => {
+    // notice that no relevant environment variables are defined here.
+    const ret = await ZapierAction.available(veryFakeOrganization);
+    expect(ret.result).toBe(false);
+  });
+
+  it("should report available if minimum environment vars ARE set", async () => {
+    process.env.BASE_URL = "https://spoke.marxwasright.com";
+
+    process.env.ZAPIER_WEBHOOK_URL =
+      "https://hooks.zapier.com/hooks/catch/9728183/buwz0q0/";
+    const ret = await ZapierAction.available(veryFakeOrganization);
+    expect(ret.result).toBe(true);
+
+    delete process.env.ZAPIER_WEBHOOK_URL;
+
+    process.env.ZAPIER_ACTION_URL =
+      "https://hooks.zapier.com/hooks/catch/9728183/buwz0q0/";
+    const ret2 = await ZapierAction.available(veryFakeOrganization);
+    expect(ret2.result).toBe(true);
+  });
+
+  describe("#onTagUpdate", async () => {
+    it("should bail if ZAPIER_WEBHOOK_URL is undefined", async () => {
+      process.env.ZAPIER_WEBHOOK_URL = undefined;
+      const ret = await ZapierAction.onTagUpdate(null, null, null, null, null);
+      expect(log.info.mock.calls[0][0]).toEqual(
+        "ZAPIER_WEBHOOK_URL is undefined. Exiting."
+      );
+    });
+
+    it("should craft the expected payload and submit it to the configured endpoint", async () => {
+      process.env.BASE_URL = "https://spoke.marxwasright.org";
+      process.env.ZAPIER_WEBHOOK_URL =
+        "https://hooks.zapier.com/hooks/catch/9728183/buwz0q0/";
+
+      // sample input data
+
+      let tags = [
+        {
+          id: "5",
+          name: "four"
+        },
+        {
+          id: "6",
+          name: "five"
+        }
+      ];
+
+      let contact = {
+        id: 59848,
+        campaign_id: 62,
+        assignment_id: 95,
+        external_id: "1",
+        first_name: "Preston",
+        last_name: "Maness",
+        cell: "+15129551048",
```

Comment:
Please change the cell to a fake one (regardless of whether this is a real number) -- if you would like your name in the tests, I'm fine with that, but I'd prefer an obviously fake name.
For fake phone numbers examples are in:
__test__/test_data/female_scientists.csv which you can grab from -- see https://www.businessinsider.com/555-phone-number-tv-movies-telephone-exchange-names-ghostbusters-2018-3 for why these are fake.

I don't want PII or even something that could be mistaken for PII in the project.